kpi/fields/jsonschema_form_field.py

```diff
@@ -66,11 +80,20 @@ def __init__(self, *args, **kwargs):
             'uniqueItems': True,
             'items': {
                 'type': 'object',
-                'required': ['name', 'required'],
+                'required': ['name', 'required',],
                 'additionalProperties': False,
                 'properties': {
-                    'name': {'type': 'string'},
-                    'required': {'type': 'boolean'},
+                    'name': {'type': 'string',},
+                    'required': {'type': 'boolean',},
+                    'label': {
+                        'type': 'object',
+                        'uniqueItems': True,
+                        'properties': {
+                            'default': {'type': 'string',},
+                        },
+                        'required': ['default',],
+                        'additionalProperties': True,
```

Comment:
LGTM except for the trailing commas within single-line dictionaries and lists; see previous comment